api-server-nest/src/authentication/filters/remove-sensitive-data-for-non-admins.ts

```diff
@@ -0,0 +1,52 @@
+import {Request} from "express";
+import {CurrentEvent} from "../../current-events/models/current-event.model";
+import {Event} from '../../events/models/event.model'
+import isAdminUser from "../is-admin-user";
+import {Model} from "sequelize-typescript";
+
+type GenericEvent = Event | CurrentEvent
+type GenericEventList = Event[] | CurrentEvent[]
+
+export async function removeSensitiveDataForNonAdmins(
+    request: Request,
+    events: GenericEvent | GenericEventList
+): Promise<GenericEvent | GenericEventList> {
+    const isAdmin = await isAdminUser(request);
```

Comment:
This bit here is probably not needed for the current implementation, but I'm a little worries about removing it in this pass.

It seems that this was written such that we would just apply this filter to all event endpoints and then skip it for non-admins, more like how the other api worked